packages/validator/src/services/attestationDuties.ts

```diff
@@ -60,6 +62,7 @@ export class AttestationDutiesService {
     clock.runEveryEpoch(this.runDutiesTasks);
     clock.runEverySlot(this.prepareForNextEpoch);
     chainHeadTracker.runOnNewHead(this.onNewHead);
+    syncingStatusTracker.runOnResynced((slot) => this.runDutiesTasks(computeEpochAtSlot(slot)));
```

Comment:
Do we wanna skip `runDutiesTasks` if slot is first slot of epoch? would avoid running duties tasks twice but in practice likley does not happen very often

Reply:
Can we wrap here? Something like
```ts
syncingStatusTracker.runOnResynced((slot) => 
  // don't run on the first slot of an epoch since we do that already
  !isFirstSlotInEpoch(slot) &&
  this.runDutiesTasks(computeEpochAtSlot(slot))
);
```

Reply:
yeah, that would be the idea, I think it's worth it to consider this case